README.md

```diff
@@ -1,2 +1,26 @@
-# base-app
-A simple JS app.
+# Awesome Tasks
+
+A simple JavaScript app that will help you getting things done.
+
+## Key concepts
+
+In order to understand the code of this simple app, is necessary to grasp these concepts:
+
+### JavaScript
+  * Base values: `String`, `Integer`, `Boolean`, undefined values
+  * Variables and functions
+  * Compound values: `Array` and `Object`
+  * Language Structure: `if`, ternary operator
```

Comment:
Yes, this is a leftover from the previous examples. We will not talk about it, I removed the ternary from the code but I didn't update the README.